.rustfmt-toml

```diff
@@ -0,0 +1,6 @@
+# These formatting rules to try conform the Substrate style guidelines:
+# >   https://wiki.parity.io/Substrate-Style-Guide
+
+reorder_imports = true
+hard_tabs = true
```

Comment:
The name of this file should be `.rustfmt.toml`, the link from above returns a 404 error, and using `hard_tabs = true` is stupid. I would remove this file and keep the default format settings.

Reply:
cool, just copied over